app/client/templates/views/dashboard.js

```diff
@@ -18,8 +18,8 @@ Template['views_dashboard'].helpers({
 
     @method (wallets)
     */
-    'wallets': function(disabled){
-        var wallets = Wallets.find({disabled: disabled}, {sort: {creationBlock: 1}}).fetch();
+    'wallets': function(){
+        var wallets = Wallets.find({disabled: false}, {sort: {creationBlock: 1}}).fetch();
```

Comment:
This needs to be changed to `Wallets.find({disabled: {$exists: false}}, {sort: {creationBlock: 1}}).fetch();`.
But we should show also disabled wallets. Then we could simply do `Wallets.find({}, {sort: {creationBlock: 1}}).fetch();`